### ISSUE TITLE
Tab Alignment ist not working for Kendo TabStrip

### ISSUE THREAD

**Issue by agrt56** [telerik/kendo-angular#3498]
**Describe the bug**
Kendo Tabstrip "Tab Alignment" feature is not working in the newest kendo ui release.

**To Reproduce**

1. Set the `tabAlignment` property to "end" or "center"

Try it out:
https://www.telerik.com/kendo-angular-ui/components/layout/tabstrip/tabs/#toc-tab-alignment

**Expected behavior**
The tabs should be rendered at the end or in the center.

**Browser**
 - Windows
 - Chrome
 - Version 95.0.4638.69

**Comment by agrt56** [comment]
Workaround (example for `tabAlignment="end"`):

```
kendo-tabstrip[tabalignment="end"] {
  .k-tabstrip-items-wrapper {
    justify-content: flex-end;
  }
}
```

**Comment by Raisolution** [comment]
Hi @agrt56 ,

Thank you for reporting the problem and please excuse us for this regression.

We have published version `6.3.6` of the `kendo-angular-layout` package which contains a fix for the issue.
